enclave/core/CMakeLists.txt

```diff
@@ -212,6 +212,8 @@ if (OE_SGX)
         -nostdinc
         -fstack-protector-strong
         -fvisibility=hidden
+        # Preserve frame-pointer in Release mode to enable oe_backtrace.
+        -fno-omit-frame-pointer
```

Comment:
Doesn't all code need to be compiled with this option? Not just liboecore

Reply:
Good point. 
Adding an option here in the PUBLIC compilation options for oecore ensures that all other source files including user written ones get the option if compiled via cmake.

I have updated pkgconfig as well so that the flag is propagated for make file based workflow.